dbms/programs/server/MySQLHandler.cpp

```diff
@@ -267,15 +267,11 @@ void MySQLHandler::comPing()
     packet_sender->sendPacket(OK_Packet(0x0, client_capability_flags, 0, 0, 0), true);
 }
 
-static bool isFederatedServerSetupCommand(String query)
+static bool isFederatedServerSetupCommand(const String &query)
 {
-    if ((0 == strncasecmp("SET NAMES", query.c_str(), 9)) || (0 == strncasecmp("SET character_set_results", query.c_str(), 25))
+    return ((0 == strncasecmp("SET NAMES", query.c_str(), 9)) || (0 == strncasecmp("SET character_set_results", query.c_str(), 25))
```

Comment:
Brackets (around each condition and around the whole expression) are redundant

Reply:
fixed